appinfo.go

```diff
@@ -71,6 +74,20 @@ func getZone() string {
 	return zone
 }
 
+func getProjectNumber(projectID string) string {
+	cloudresourcemanagerService, err := cloudresourcemanager.NewService(context.Background())
```

Comment:
Can we setup this client like we do for `k8sClientSet` & `istioClientSet`? We ran into memory issues making clients like this on the fly.

I don't know how expensive these calls to get the project might be either but it could be helpful to cache these in a map so once it's looked up once we wouldn't have to do it again as it shouldn't change.

Or just set these as env_vars like we do for `GOOGLE_CLOUD_PROJECT` for the project names